testing/tests/DevExpress.ui.widgets.pivotGrid/summaryDisplayModes.tests.js

```diff
@@ -1,71 +1,69 @@
-var $ = require("jquery"),
-    summaryDisplayModes = require("ui/pivot_grid/ui.pivot_grid.summary_display_modes"),
-    applyDisplaySummaryMode = summaryDisplayModes.applyDisplaySummaryMode, // arguments: description, data
-    applyRunningTotal = summaryDisplayModes.applyRunningTotal, // arguments: description, data
-    pivotGridUtils = require("ui/pivot_grid/ui.pivot_grid.utils"),
-    summaryDictionary = summaryDisplayModes.summaryDictionary,
-    data = {
-        columns: [{
-            index: 1,
-            value: "1",
-            children: [{
-                index: 2,
-                value: "11"
-            }, {
-                index: 3,
-                value: "12"
-            }, {
-                index: 4,
-                value: "13"
-            }]
+import $ from "jquery";
+import summaryDisplayModes, { applyDisplaySummaryMode, applyRunningTotal, summaryDictionary } from "ui/pivot_grid/ui.pivot_grid.summary_display_modes"; // arguments: description, data
+import pivotGridUtils from "ui/pivot_grid/ui.pivot_grid.utils";
+
+var data = {
+    columns: [{
+        index: 1,
+        value: "1",
+        children: [{
+            index: 2,
+            value: "11"
         }, {
-            index: 5,
-            value: "2"
+            index: 3,
+            value: "12"
```

Comment:
Why did you change this data item?

Reply:
This data item is not changed. Diff tool woks bad.